config/initializers/content_security_policy.rb

```diff
@@ -10,7 +10,8 @@
   policy.img_src :self, :https, :data, "*.s3.amazonaws.com"
   policy.object_src :none
   policy.frame_ancestors :none
-  policy.script_src :self, :https, :unsafe_inline, :unsafe_eval, "*.stripe.com", "openfoodnetwork.innocraft.cloud"
+  policy.script_src :self, :https, :unsafe_inline, :unsafe_eval, "*.stripe.com", "openfoodnetwork.innocraft.cloud",
+                    "maps.googleapis.com", "maps.gstatic.com", "d2wy8f7a9ursnm.cloudfront.net"
```

Comment:
Hm, I guess that d2wy8f7a9ursnm.cloudfront.net could change without warning...

Reply:
As far as I can tell it's Bugsnag's JS endpoint and it's very stable. We've never changed it and it's the URL in their docs :man_shrugging:

Reply:
Right. Sounds relatively safe then.